### PR TITLE
[fix](inverted index)Support Chinese column name with inverted index

### PR DESCRIPTION
**Pull request by qidaye**
## Proposed changes

1. `std::string` to `std::wstring` conversion only supports ASCII characters. For non-ASCII characters, we need to use `StringUtil::string_to_wstring`
2. Fix index_tool check_terms_stats_v2 and add field info to print

Issue Number: #34118

### CODE REVIEW

**Comment by qidaye** [comment]
run buildall

**Review by zzzxl1993 — APPROVED** [review]
LGTM

**Review by airborne12 — APPROVED** [review]
LGTM

File: be/src/olap/rowset/segment_v2/inverted_index_reader.cpp [hunk 1]
```diff
@@ -353,7 +353,7 @@ Status FullTextIndexReader::query(OlapReaderStatistics* stats, RuntimeState* run
         }
 
         std::unique_ptr<lucene::search::Query> query;
-        query_info.field_name = std::wstring(column_name.begin(), column_name.end());
```
Comment (xiaokang) [inline be/src/olap/rowset/segment_v2/inverted_index_reader.cpp:356]: grep std::wstring in the code and fix all

Comment (qidaye) [reply be/src/olap/rowset/segment_v2/inverted_index_reader.cpp:356]: done

**Comment by qidaye** [comment]
run buildall

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**